-------------------------------------------------------------------------------
Test set: TestSuite
-------------------------------------------------------------------------------
Tests run: 12, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 41.687 s <<< FAILURE! - in TestSuite
automationtest.test.Verifyfalse.Checkout3[public void automationtest.test.Verifyfalse.Checkout3(java.lang.reflect.Method) throws java.lang.InterruptedException](1)  Time elapsed: 0.19 s  <<< FAILURE!
java.lang.AssertionError: expected [rr] but found [THANK YOU FOR YOUR ORDER]
	at automationtest.test.Verifyfalse.Checkout3(Verifyfalse.java:98)

